packages/web3/package.json

```diff
@@ -5,7 +5,7 @@
 	"main": "lib/index.js",
 	"repository": "https://github.com/ChainSafe/web3.js",
 	"engines": {
-		"node": ">=12.0.0"
+		"node": ">=14.0.0"
```

Comment:
Some packages have an npm min version, and some - not. I think it will be great to have the same config in all packages. Also, maybe we should add engine config to all packages. What do you think?
```json
"engines": {
		"node": ">=14",
		"npm": ">=6.12.0"
	},
```

Reply:
Sounds like a good suggestion to me.